explainer.md

```diff
@@ -365,41 +365,7 @@ function checkARSupport() {
-
-An `XRPresentationContext` can only be supplied imagery by an `XRSession`, though the exact behavior depends on the scenario in which it's being used. The context is associated with a session by setting the `XRRenderState`'s `outputContext` to the desired `XRPresentationContext` object. An `XRPresentationContext` cannot be used with multiple `XRSession`s simultaneously, so when an `XRPresentationContext` is set as the `outputContext` for a session's `XRRenderState`, any session it was previously associated with will have it's `renderState.outputContext` set to `null`.
-
-### Mirroring
-
-On desktop devices, or any device which has an external display connected to it, it's frequently desirable to show what the user in the headset is seeing on the external display. This is usually referred to as mirroring.
-
-In order to mirror WebXR content to the page, the session's `renderState.outputContext` must be set to a `XRPresentationContext`. Once a valid `outputContext` has been set any content displayed on the headset will then be mirrored into the canvas associated with the `outputContext`.
-
-When mirroring only one eye's content will be shown, and it should be shown without any distortion to correct for headset optics. The UA may choose to crop the image shown, display it at a lower resolution than originally rendered, and the mirror may be multiple frames behind the image shown in the headset. The mirror may include or exclude elements added by the underlying XR system (such as visualizations of room boundaries) at the UA's discretion. Pages should not rely on a particular timing or presentation of mirrored content, it's really just for the benefit of bystanders or demo operators.
-
-The UA may also choose to ignore the `outputContext` on systems where mirroring is inappropriate, such as devices without an external display like mobile or all-in-one systems.
-
-```js
-function beginXRSession() {
-  let mirrorCanvas = document.createElement('canvas');
-  let mirrorCtx = mirrorCanvas.getContext('xrpresent');
-  document.body.appendChild(mirrorCanvas);
-
-  navigator.xr.requestSession('immersive-vr')
-      .then((session) => {
-        // A mirror context isn't required to render, so it's not necessary to
-        // wait for the updateRenderState promise to resolve before continuing.
-        // It may mean that a frame is rendered which is not mirrored.
-        session.updateRenderState({ outputContext: mirrorCtx });
-        onSessionStarted(session);
-      })
-      .catch((reason) => { console.log("requestSession failed: " + reason); });
-}
-```
-
-### Inline sessions
+## Inline sessions
 
 There are several scenarios where it's beneficial to render a scene whose view is controlled by device tracking within a 2D page. For example:
 
```

Comment:
The Tango link on line 373 is no longer live.

Reply:
Thanks for pointing that out! Replaced with ARCore/ARKit links